### DISCUSSION TITLE
Missing info on - Proxmox VE Cron LXC Updater description

### DISCUSSION THREAD

**Discussion by Wolffoxfangs** [community-scripts/ProxmoxVE#251] [Q&A]
I was perusing the community site and I notice that the description reads 

> Description
> 
> This script will add/remove a crontab schedule that updates all LXCs every Sunday at midnight. To exclude LXCs from updating, edit crontab (crontab -e) and add CTID as shown in the example below:

Missing the example itself. 

I did see a full example on ttecks old site that might be good to update the descriptions example.

> This script will add/remove a crontab schedule that updates all LXCs every Sunday at midnight.
> 
> To exclude LXCs from updating, edit crontab (crontab -e) and add CTID as shown in the example (-s 103 111)
> 
>` example: 0 0 * * 0 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs-cron.sh)" -s 103 111 >>/var/log/update-lxcs-cron.log 2>/dev/null `

![Community cron missing example](https://github.com/user-attachments/assets/ed6c2811-f880-4ad6-a749-88bb4ae6ac21)

![TTeck Cron w example](https://github.com/user-attachments/assets/d005fb62-e823-4426-8a8b-e519447b50e6)

**Comment by havardthom** [comment]
Fixed in https://github.com/community-scripts/ProxmoxVE/pull/378
